source/view.c

```diff
@@ -794,6 +807,66 @@ void CalcGunAngle (void)
 	cl.viewent.angles[PITCH] = -1 * ((r_refdef.viewangles[PITCH] + pitch) - (angledelta((r_refdef.viewangles[PITCH] + pitch) + OldPitchTheta ) * 0.2));
 
 	//cl.viewent.angles[PITCH] = - (r_refdef.viewangles[PITCH] + pitch);
+#else
```

Comment:
Can you clean up your comments here and add some details that describes this chunk?